tests/data/dataset_test.py

```diff
@@ -38,22 +38,22 @@ def test_as_arrays(self):
         dataset.index_instances(self.vocab)
         padding_lengths = dataset.get_padding_lengths()
         arrays = dataset.as_arrays(padding_lengths)
+        text1 = arrays["text1"]["tokens"]
+        text2 = arrays["text2"]["tokens"]
 
-        text1 = arrays["text1"][0]
-        text2 = arrays["text2"][0]
         numpy.testing.assert_array_almost_equal(text1, numpy.array([[2, 3, 4, 5, 6],
                                                                     [1, 3, 4, 5, 6]]))
         numpy.testing.assert_array_almost_equal(text2, numpy.array([[2, 3, 4, 1, 5, 6],
                                                                     [2, 3, 1, 0, 0, 0]]))
 
     def get_dataset(self):
         field1 = TextField(["this", "is", "a", "sentence", "."],
-                           [token_indexers["single id"]("words")])
+                           {"tokens": token_indexers["single id"]()})
```

Comment:
You could just define this once, at the top of the method, and use the variable in these four lines.